src/bp/core/services/telemetry/sdk-methods.ts

```diff
@@ -0,0 +1,188 @@
+import { parse } from '@babel/parser'
+import traverse from '@babel/traverse'
+import LicensingService from 'common/licensing-service'
+import { buildSchema } from 'common/telemetry'
+import Database from 'core/database'
+import { TelemetryRepository } from 'core/repositories/telemetry'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import _ from 'lodash'
+import ms from 'ms'
+
+import { GhostService } from '..'
+import { BotService } from '../bot-service'
+import { JobService } from '../job-service'
+
+import { BotHooks, getHooksLifecycle, Hook } from './hooks'
+import { TelemetryStats } from './telemetry-stats'
+
+const PARSE_CONFIG = { allowReturnOutsideFunction: true }
+
+interface UsageParams {
+  botId?: string
+  lifecycle?: string
+  enabled?: Boolean
+}
+
+interface Usage {
+  namespace: string
+  function: string
+  count: number
+}
+
+interface ParsedFile extends UsageParams {
+  fileName: string
+  usages: Usage[]
+}
```

Comment:
We use types when we need to extend an interface instead:
```js

type ParsedFile = {
 fileName: string
  usages: Usage[]
} & UsageParams
```

Reply:
fixed: ea8ccef7154fbde172eb306b7fa7e0f733e5592f